### PR TITLE
updated sec 6.3

### PR DESCRIPTION
**Pull request by JohanBester**
Hello Folks,

I updates section 6.3 according to the notes in ticket #296

Kindly please review these changes and comment as needed.

Thanks!

### CODE REVIEW

File: Section06/6.3-SPAState.md [hunk 1]
```diff
@@ -81,43 +74,23 @@ So, our **model** is passed into and rendered by our **view** which then handles
...
+5. Inside `store/index.js`, let's aggregate each of our modules with one-line import/export statements.
 
    ```javascript
 
    export { default as Mod1 } from "./Mod1";
    export { default as Mod2 } from "./Mod2";
    ...
 
-7. Now our state modules are prepped and collected and ready to import. In our root `index.js`, let's `import` our state modules as a module Object named `store`.
+5. Now our state modules are prepped and collected and ready to import. In our root `index.js`, let's `import` our state modules as a module Object named `store`.
```
Comment (matsinet) [inline Section06/6.3-SPAState.md:85]: this should be item 6

**Review by matsinet — CHANGES_REQUESTED** [review]

**Review by mjwebb1993 — CHANGES_REQUESTED** [review]
I can touch up on this in the slack channel, but would it be possible to implement an additional change to this PR? I think the very last step in 6.3, which says to remove the following code from the router hook:
`document.querySelectorAll("nav a").forEach((navLink) =>
  navLink.addEventListener("click", (event) => {
    event.preventDefault();
    render(state[event.target.title]);
  })
);`

I don't believe this code is necessary as we did not have this in our code, at least not at the beginning of the night

File: Section06/6.3-SPAState.md (hunk 1 above)
Comment (JohanBester) [reply Section06/6.3-SPAState.md]: Hi Matt, thanks for catching that. I fixed up the order and made the number changes.  Thank you.

**Comment by JohanBester** [comment]
Hello Gents,
I have made the changes you suggested.

Please review those and let me know if that works for you.
I think I got it right this time 😁

**Review by matsinet — DISMISSED** [review]

**Review by mjwebb1993 — APPROVED** [review]

**Pull request merged**